docker-compose.yml

```diff
@@ -1,14 +1,11 @@
 version: '3.2'
 services:
   server:
-    container_name: "song-server-$VERSION"
-    image: "overture/song-server:$VERSION"
+    container_name: "song-server-local"
+    image: "overture/song-server:local"
     build:
       context: ./
```

Comment:
you can create a docker file for local run (without doing maven build inside it, just copy the local built artifacts) and a docker file for the release image (this PR), because I'm noticing you are forcing the compose to always be local. this could be another issue if you see that local work flow becomes slow with docker